### ISSUE TITLE
running 2 SSL listeners on ELB with different ports

### ISSUE THREAD

**Issue by jelbatnigi** [aws/aws-sdk-java#1176]
I have a springboot web application that is running in an AWS EC2. The application is running behind a classic ELB. I am using HTTPS between the client and the ELB so traffic coming in on port 443 is being routed to port 8080 I have deployed the certificate to the ELB.

In the same application I have an embedded ActiveMQ running on port 61616. It is running as part of the JVM. Clients connect to it using TCP (TCP://domain.com:61616). I want the client to connect to my AMQ using SSL similar to the way they connect to the application (through HTTPS). I have added a listener to the ELB where the client connects to the ELB using SSL (SSL://domain.com:61616) and the ELB routes to the internal port using TCP and I have deployed the same certificate to the ELB as the one I used for the application. for example here is what I have: 
ELB protocol,      ELB port,      instance protocol,      instance port,      cypher,       SSL certificate
     HTTPS,               443,                      HTTP,                  8080,              change,            ssl.cert
     SSL,                   61616,                   TCP,                    61617,            change,            ssl.cert

Basically I want to use SSL between the client and the ELB and TCP from ELB to the instance. Why this doesn't work? when I try to connect using openssl

openssl s_client -connect domain.com:61616

I get the following:
`CONNECTED(00000003)
write:errno=104
no peer certificate available
No client certificate CA names sent
SSL handshake has read 0 bytes and written 247 bytes
New, (NONE), Cipher is (NONE) Secure Renegotiation IS NOT supported Compression: NONE Expansion: NONE ---`
 Why I cannot use a different port other than 443 to use SSL?

**Comment by varunnvs92** [comment]
This doesn't looks like an SDK issue but rather an issue with security certificates related to ELB and EC2. This channel is for tracking bugs and feature requests in AWS SDK for Java. See [Getting Help](https://github.com/aws/aws-sdk-java#getting-help) Section.

I am sorry that I could not help you as I don't have expertise in networking and SSL. I found a similar [issue](https://stackoverflow.com/questions/24457408/openssl-command-to-check-if-a-server-is-presenting-a-certificate) on stack overflow, this might help. Please redirect this question to stack overflow or AWS Service forums for better support. Thank you.
